webapp/playbooks/src/components/backstage/channel_selector.tsx

```diff
@@ -32,7 +32,6 @@ export interface Props {
 }
 
 const getAllPublicChannelsInTeam = (teamId: string) => createSelector(
-    'getAllPublicChannelsInTeam',
```

Comment:
Why these removals? Don't we want to track these?

Reply:
We do some tracking in webapp, but not in plugins. Plan with this custom reselect was always to measure (even though with false positive results it may show) on webapp selectors